util/__init__.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+from libtbx.math_utils import round2
```

Comment:
Do we have a good case for retaining this?

Reply:
... probably no?